roles/openshift_hosted/tasks/registry/secure.yml

```diff
@@ -20,11 +20,11 @@
   failed_when: false
 
 - name: Retrieve registry service IP
-  command: >
-    {{ openshift.common.client_binary }} get service docker-registry
-    -o jsonpath='{.spec.clusterIP}'
-    --config={{ openshift_hosted_kubeconfig }}
-    -n default
+  oc_service:
+    namespace: default
+    name: docker-registry
+    state: list
+    kubeconfig: "{{ openshift_hosted_kubeconfig }}"
```

Comment:
The same here, can we be sure ``{{ openshift_hosted_kubeconfig }}`` is always ``/etc/origin``?